widgets/blueprints/src/TerraformModal.tsx

```diff
@@ -50,6 +50,7 @@ function getDynamicTableDropdown(options: DropdownProps['options']) {
 }
 
 const cloudifyResourceRegexp = /^[a-zA-Z][a-zA-Z0-9._-]*$/;
+const cloudifyStaticValueRegexp = /^[a-zA-Z0-9._-]*$/;
```

Comment:
Actually I think the better name would be something relating to Terraform as static value is directly passed to `resource_config` property in `cloudify.nodes.terraform.Module` node in the resulting blueprint. In the other cases we are using the value as Cloudify resource (either secret key or input name).

```suggestion
const terraformResourceRegexp = /^[a-zA-Z0-9._-]*$/;
```

Reply:
Hmm, that might be a valuable hint
But I think that this perspective is related to the other parts of the system     

It may get confusing, when somebody would have to get into this component to modify the validation for the `static values` and see that under the hood they are just a terraform resource   

I think that the knowledge about what is happening under the hood is valuable, but I don't think that connecting the names across distant system parts may be a good option 🤔 

What do you think? 🤔

Reply:
On one hand yes (good argument with connecting names across distant parts of the system), on the other no (the whole feature is about converting cloudify resources into terraform resources). 

Anyways, I'm OK with keeping `StaticValue` part, but I think we shouldn't use `cloudify` prefix in the name, let's call it just `staticValueRegexp`.

Reply:
Okay, I'm perfectly fine with it 😉 
I'll add it with the next commit

Thanks for showing your perspective upon that matter! 🎉

Reply:
Commit with the changes has arrived 🎉